examples/brainfuck/src/main.rs

```diff
@@ -107,6 +107,7 @@ fn codegen<'a, 'ctx>(ops: &[Op], context: &'a gccjit::Context<'ctx>) -> bool {
     let mut current_block = brainf_main.new_block("entry_block");
     // now we have to zero out the giant buffer we just allocated on the stack.
     let zero_access = context.new_array_access(None, array.to_rvalue(), context.new_rvalue_zero(int_ty));
+    // a call to context.new_call will give us warning
```

Comment:
I would only keep this comment, so please remove the others.

And please edit the comment to something like this:
```suggestion
    // A function call that is done for its side effects must be sent to add_eval.
```

Reply:
Alright